app.py

```diff
@@ -12,12 +14,11 @@
 @app.route('/data.json')
 def data():
     access_token = app.config['ACCESS_TOKEN']
-    users = app.config['USERS']
     start_date = valid_date(app.config['START_DATE'])
     end_date = valid_date(app.config['END_DATE'])
 
     contribs = Contributions(GithubStats(access_token=access_token))
-    stats = contribs.leaderboard(users, start_date, end_date)
+    stats = contribs.leaderboard(load_users(), start_date, end_date)
```

Comment:
stick to `stats = contribs.leaderboard(users, start_date, end_date)` 

and have users = load_users(app.config['user_file_path'])